src/app/modules/group/components/manager-permission-dialog/manager-permission-dialog.component.html

```diff
@@ -25,50 +25,69 @@
         This panel lets you select what you allow the selected user or group of users to do on the current group and its descendants.
       </p>
```

Comment:
**(small) Bug** In the permission edit dialog, when a collapsible section is uncollapsed, the cursor when hovering the header should be a pointer

**Steps to reproduce**
1. Watch [this group](https://dev.algorea.org/branch/permissions-dialog-refact/en/#/groups/by-id/4761034848060385407;path=/details)
2. Click on the chapter link that appears in the overlay
3. Navigate to Progress > Chapter view
4. Click on first score you see (ie annabelle4) and click "Access" to open permission edit dialog

Reply:
fixed